components/eamxx/src/physics/ace/eamxx_ace_process_interface.cpp

```diff
@@ -0,0 +1,856 @@
+  add_field<Updated>("sfc_flux_dir_vis", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("sfc_flux_dif_nir", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("sfc_flux_dif_vis", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("sfc_flux_sw_net", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("sfc_flux_lw_dn", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("precip_liq_surf_mass", sc_layout_2d_scalar, kg / m2, m_sc_grid->name());
+  add_field<Updated>("precip_ice_surf_mass", sc_layout_2d_scalar, kg / m2, m_sc_grid->name());
+
+  // Stuff ACE can (but doesn't have to) use from the sc_import
+  // NOTE: "Updated" seems to be necessary for the ll-sc syncing in initialize_impl
+  // TODO: ask Luca to take a look into this?
+  add_field<Updated>("sfc_alb_dir_vis", sc_layout_2d_scalar, nondim, m_sc_grid->name());
+  add_field<Updated>("sfc_alb_dir_nir", sc_layout_2d_scalar, nondim, m_sc_grid->name());
+  add_field<Updated>("sfc_alb_dif_vis", sc_layout_2d_scalar, nondim, m_sc_grid->name());
+  add_field<Updated>("sfc_alb_dif_nir", sc_layout_2d_scalar, nondim, m_sc_grid->name());
+  add_field<Updated>("surf_lw_flux_up", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("surf_sens_flux", sc_layout_2d_scalar, W / m2, m_sc_grid->name());
+  add_field<Updated>("surf_evap", sc_layout_2d_scalar, kg / m2 / s, m_sc_grid->name());
+  add_field<Updated>("surf_mom_flux", sc_layout_2d_vector, N / m2, m_sc_grid->name());
+  add_field<Updated>("surf_radiative_T", sc_layout_2d_scalar, K, m_sc_grid->name());
+  add_field<Updated>("T_2m", sc_layout_2d_scalar, K, m_sc_grid->name());
+  add_field<Updated>("qv_2m", sc_layout_2d_scalar, kg / kg, m_sc_grid->name());
+  add_field<Updated>("wind_speed_10m", sc_layout_2d_scalar, m / s,m_sc_grid->name());
+  add_field<Updated>("snow_depth_land", sc_layout_2d_scalar, m,m_sc_grid->name());
+  add_field<Updated>("ocnfrac", sc_layout_2d_scalar, nondim, m_sc_grid->name());
+  add_field<Updated>("landfrac", sc_layout_2d_scalar, nondim,m_sc_grid->name());
+  add_field<Updated>("icefrac", sc_layout_2d_scalar, nondim, m_sc_grid->name());
+
+  // Create two big fields for input and output for ease (can remove later)
+  FieldLayout scalar3d_39x180x360_mid{{CMP, CMP, LAT, LON}, {m_batch, m_in_ch, m_height, m_width}};
+  FieldLayout scalar3d_44x180x360_mid{{CMP, CMP, LAT, LON}, {m_batch, m_out_ch, m_height, m_width}};
+  m_input_field = Field(FieldIdentifier("InField", scalar3d_39x180x360_mid, nondim, m_ll_grid->name()));
+  m_output_field = Field(FieldIdentifier("OutField", scalar3d_44x180x360_mid, nondim, m_ll_grid->name()));
+  m_input_field.allocate_view();
+  m_output_field.allocate_view();
+}
+
+void ACE::initialize_impl(const RunType /* run_type */) {
+  // Output configuration values
+  std::cout << "ACE::initialize_impl: Checkpoint Path: " << m_checkpoint_path << std::endl;
```

Comment:
Let's replace these calls with calls to `log`. E.g., `log(LogLevel::info, ACE::initialize_impl: Checkpoint Path: " + m_checkpoint_path);`.